src/components/views/rooms/MessageComposer.tsx

```diff
@@ -258,7 +259,7 @@ export default class MessageComposer extends React.Component<IProps, IState> {
 
     private renderPlaceholderText = () => {
         if (this.props.replyToEvent) {
-            const replyingToThread = this.props.relation?.rel_type === RelationType.Thread;
+            const replyingToThread = this.props.relation?.rel_type === THREAD_RELATION_TYPE.name;
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?